packages/turf-line-chunk/index.js

```diff
@@ -0,0 +1,94 @@
+var along = require('@turf/along');
+var lineDistance = require('@turf/line-distance');
+var helpers = require('@turf/helpers');
+var flatten = require('@turf/flatten');
+var linestring = helpers.lineString;
+var point = helpers.point;
+var fc = helpers.featureCollection;
+
+/**
+ * Divides a {@link LineString} into chunks of a specified length.
+ * If the line is shorter than the segment length then the orginal line is returned.
+ * @name lineChunk
+ * @param {Feature<LineString>} line the line to split
```

Comment:
This module should support `{FeatureCollection|Feature<LineString|MultiLineString>}` as input